dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -0,0 +1,19 @@
+class FeaturedProjectsController < ApplicationController
+  def create
+    _, channel_id = storage_decrypt_channel_id(featured_project_params[:project_id])
+    @featured_project = FeaturedProject.create({storage_app_id: channel_id, who_featured_user_id: current_user.id})
+    @featured_project.save!
+  end
+
+  def destroy_featured_project
```

Comment:
`destroy` See comments in `routes.rb` below. 

 1. `destroy` is the default delete action created by the `resources` routing macro. No need to write more code and break convention.
 2. `_featured_project` is redundant in the `FeaturedProjectsController`

Reply:
@aoby , in case this was missed, the id we're using here is the project_id, which is different from the featured_project_id that would implicitly be used in the rails routes generated by the `resource` command.

Reply:
Ah yes, I did miss that. Thanks for the clarification! 

In that case, how about `destroy_by_project_id` or something that is more clearly not the default destroy (by featured project id) action?